chat/rich-links/README.md

```diff
@@ -0,0 +1,19 @@
+# Rich Links
```

Comment:
Can you add the props to the README?

Reply:
Will add once I've resolved the anchorProps/Card/as TS error